services/lnd/grpc.js

```diff
@@ -841,9 +842,11 @@ class LndGrpcWrapper extends EventEmitter {
         }
       });
       sentPayment.on('status', status => {
+        this.emit('sendPayment.status', status);
         grpcLog.info('received sendPayment status %O', status);
       });
       sentPayment.on('error', error => {
+        this.emit('sendPayment.error', error);
         if (error && error.message.includes('amount must be specified')) {
```

Comment:
how about just emitting on successful payment as that's the only time a balance would be updated and also removing the status/error since we don't currently use it?